lib/elixir/lib/kernel/cli.ex

```diff
@@ -146,7 +146,8 @@ defmodule Kernel.CLI do
   # Parse shared options
 
   defp parse_shared([opt|_t], _config) when opt in ["-v", "--version"] do
-    IO.puts "Elixir #{System.version}"
+    IO.puts "Erlang/OTP #{:erlang.system_info(:otp_release)}"
```

Comment:
Let's do: `IO.puts :erlang.system_info(:system_version)` since it shows more information regarding the environment.